beacon_node/beacon_chain/src/lib.rs

```diff
@@ -1,4 +1,3 @@
-#![recursion_limit = "128"] // For lazy-static
```

Comment:
Note we don't need to set the recursion limit to 128 because that's the default now